config/application.rb

```diff
@@ -64,12 +64,14 @@ class Application < Rails::Application
     config.active_record.encryption.key_derivation_salt = ENV.fetch('ENCRYPTION_KEY_DERIVATION_SALT')
 
     # TODO(ezekg) Remove these once our data is migrated
+    config.active_record.encryption.hash_digest_class                             = OpenSSL::Digest::SHA256
+    config.active_record.encryption.support_sha1_for_non_deterministic_encryption = true
+    config.active_record.encryption.store_key_references     = true
```

Comment:
Remove this.

Reply:
Shouldn't have added this. Broke license key lookups, since the lookups were now including the key reference. 🤦‍♂️